content/documentation/download.md

```diff
@@ -52,8 +52,8 @@ The following requirements apply to the installation from source:
 In order to build and install IPFS Cluster in Unix systems follow the steps:
 
 ```
-git clone https://github.com/ipfs/ipfs-cluster.git $GOPATH/github.com/ipfs/ipfs-cluster
-cd $GOPATH/github.com/ipfs/ipfs-cluster
+go get -u -d github.com/ipfs/ipfs-cluster
```

Comment:
Sorry, this is wrong. This will still download all deps, even if it won't install the application. We don't need to download all deps because we use `gx` for that.

Reply:
I will just add src then

Reply:
@hsanjuan Changed. Sorry for force push.